File: LeetCode/2535_Difference_Between_Element_Sum_and_Digit_Sum_of_an_Array/solution.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var differenceOfSum = function(nums) {
    let elemsum=0;
    let digsum=0;
    for(let i=0;i<nums.length;i++){
        elemsum+=nums[i];
        let a = nums[i];
        while(a>0){
            let m=a%10;
            digsum+=m;
            a=Math.floor(a/10);
        }
    }
    return Math.abs(elemsum-digsum);
};

File: LeetCode/2535_Difference_Between_Element_Sum_and_Digit_Sum_of_an_Array/solution.cpp
class Solution {
public:
    int differenceOfSum(vector<int>& nums) {
        int elemsum = 0;
        int digsum = 0;
        for (int i = 0; i < nums.size(); i++) {
            elemsum += nums[i];
            int a = nums[i];
            while (a > 0) {
                int m = a % 10;
                digsum += m;
                a /= 10;
            }
        }
        return abs(elemsum - digsum);
    }
};